CONTRIBUTING.md

```diff
@@ -14,47 +14,56 @@ Remember that if something here doesn’t make sense, you can propose a change t
+- `framework-integration-tests` - Implements integration tests for all supported vendors. Tests are run on real infraestructure using the same mechanisms than a production application. This package `src` folder includes a synthetic Booster application that can be deployed to a real provider for testing purposes.
+- `framework-provider-aws` - Implements all the required adapters to make the booster core run on top of AWS technologies like Lambda and DynamoDB using the AWS SDK under the hoods.
+- `framework-provider-aws-infrastructure` - Implements all the required adapters to allow Booster applications to be deployed to AWS using the AWS CDK under the hoods.
+- `framework-provider-local` - Implements all the required adapters to run the Booster application on a local express server to be able to debug your code before deploying it to a real cloud provider.
+- `framework-provider-local-infrastructure` - Implements all the required code to run the local development server.
+- `framework-types` - This package defines types that the rest of the project will use. This is useful for avoiding cyclic dependencies. Note that this package should not contain stuff that are not types, or very simple methods related directly to them, i.e. a getter or setter. This package defines the main booster concepts like:
   - Entity
   - Command
   - etc…
-- `core` - This one is the one that contains all the logic directly tied to the Booster architecture. Stuff like the generation of the config or the handler wrappers are defined here. This package uses the provider packages dynamically.
-- `aws` - The first provider package, which acts as an adapter between the core and the AWS SDK (more concretely, the CDK library). All stuff that is directly tied to AWS go here.
-- `cli` - You guessed it! This package is the boost command-line tool, it interacts only with the core package in order to load the configuration.
-- `example` - An example project that is updated with the changes added to the project. It is useful to have this package to perform pre-release manual smoke tests.
 
 ## How Can I Contribute?
 
+Contributing to an open source project is never just a matter of code, you can help us significantly by just using Booster and interacting with our community. Here you'll find some tips on how to do it effectively.
+
 ### Reporting Bugs
 
-When you are creating a bug report, please include as many details as possible. Fill out the required template, the information it asks for helps us resolve issues faster.
+When you are creating a bug report, please include as many details as possible. Fill out the required template, the information requested helps us to resolve issues faster.
```

Comment:
Despite the note below, shall we encourage people to first attempt finding similar issues?

Reply:
I don't think it is needed, if a bug is duplicated it might be marked as a duplicate, and sometimes it might ocurr that an issue is old and buried, but by creating a new one it gets "refreshed". Also, we should use StaleBot for stale issues

Reply:
Good point!

```suggestion
Before creating a bug report, please search for similar issues to make sure that they're not already reported. If you don't find any, go ahead and create an issue including as many details as possible. Fill out the required template, the information requested helps us to resolve issues faster.
```